docs/js/controls-toggles.js

```diff
@@ -0,0 +1,12 @@
+export default 
+  window.requestAnimationFrame(() => {
+    let slider = document.querySelector("#slider1");
+    let counter = document.querySelector(".a-slider--value");
+
+    slider.addEventListener("input", function (){
+      let sliderValue = slider.value;
+
+      slider.style.setProperty("--slider-percentage", sliderValue + "%");
+      counter.innerText = slider.value + " coins";
```

Comment:
We can use the `sliderValue` here to avoid relying on DOM again.

```suggestion
      counter.innerText = sliderValue + " coins";
```

Reply:
And maybe use backticks

```suggestion
      counter.innerText = `${sliderValue} coins`;
```

Reply:
No idea why I created a variable for the slider value and then selected it again lol

Reply:
d762264